### PR TITLE
Fix security camera console not following moving borgs

### PR DESCRIPTION
**Pull request by Timberpoes**
## About The Pull Request

Camera consoles don't tracking moving cameras.

If the camera's turf changes between ui_updates, the camera now follows.

It's not perfect, but it gets the job done for what is ostensibly a niche feature and basically feels like any other tracking code.

## Why It's Good For The Game

Fixes an oversight in #52767

## Changelog
:cl:
fix: The security camera console will now correctly track when viewing moving cyborg cameras.
/:cl:

### CODE REVIEW

**Review by stylemistake — DISMISSED** [review]

File: code/game/machinery/computer/camera.dm [hunk 1]
```diff
@@ -60,9 +62,13 @@
...
 	// Update UI
 	ui = SStgui.try_update_ui(user, src, ui)
-	// Show static if can't use the camera
-	if(!active_camera?.can_use())
+
+	// Update the camera, showing static if we can't use it and updating data if the location has moved.
+	if(active_camera)
+		update_active_camera_screen()
+	else
 		show_camera_static()
```
Comment (stylemistake) [inline code/game/machinery/computer/camera.dm:70]: I don't like that you replaced `if(!active_camera?.can_use())` with two if statements (here and one in the proc). It only adds more loc and complexity. Just revert to how it was before (with .?)

**Review by stylemistake — CHANGES_REQUESTED** [review]
Address travis errors.

File: code/game/machinery/computer/camera.dm (hunk 1 above)
Comment (Timberpoes) [reply code/game/machinery/computer/camera.dm]: Change made.

**Comment by Timberpoes** [comment]
> 
> 
> Address travis errors.

Providing travis works, I cleared up the unused testing var.

**Review by stylemistake — APPROVED** [review]

**Pull request merged**